tests/components/mqtt/test_discovery.py

```diff
@@ -933,15 +923,15 @@ async def test_cleanup_device_multiple_config_entries(
 
 async def test_cleanup_device_multiple_config_entries_mqtt(
     hass: HomeAssistant,
-    device_reg: dr.DeviceRegistry,
-    entity_reg: er.EntityRegistry,
+    device_registry: dr.DeviceRegistry,
+    entity_registry: er.EntityRegistry,
     mqtt_mock_entry_no_yaml_config: MqttMockHAClientGenerator,
 ) -> None:
     """Test discovered device is cleaned up when removed through MQTT."""
     mqtt_mock = await mqtt_mock_entry_no_yaml_config()
     config_entry = MockConfigEntry(domain="test", data={})
     config_entry.add_to_hass(hass)
-    device_entry: dr.DeviceEntry | None = device_reg.async_get_or_create(
+    device_entry = device_registry.async_get_or_create(
```

Comment:
Not related, but showed up in my conflict. The typing was incorrect.

Reply:
Nice catch